integration-tests/src/test/resources/OperatorIT.properties

```diff
@@ -5,5 +5,5 @@ baseDir=/scratch
 #wls admin user
 username=weblogic
 password=welcome1
-maxIterationsPod=25
-waitTimePod=10
\ No newline at end of file
+maxIterationsPod=50
```

Comment:
why do you want to increase the iterations? If something is wrong, tests takes long time to return.

Reply:
reverted back